src/platform/forms/save-in-progress/SaveStatus.jsx

```diff
@@ -71,6 +71,7 @@ function SaveStatus({
 SaveStatus.propTypes = {
   form: PropTypes.object.isRequired,
   isLoggedIn: PropTypes.bool.isRequired,
+  formConfig: PropTypes.object,
```

Comment:
Can we get a `.shape().isRequired` here too?

Reply:
Fixed.